app/models/comment.rb

```diff
@@ -1,5 +1,5 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
```

Comment:
- [ ] This is creating a problem while creating a comment, as if you want to give column name as `author_id`, you have to change in migrate file as well with the `author` instead of `user` where you have added the reference of user to the comments table.

![Screenshot from 2022-03-31 03-52-08](https://user-images.githubusercontent.com/22434660/160941624-a1cc9c47-7c53-4f4a-a701-ab091b1adef9.png)

Reply:
Many thanks for this wonderful review I have made the changes 😄